hawkbit-extended-runtimes/hawkbit-update-server-azure/src/main/java/org/eclipse/hawkbit/app/AzureUI.java

```diff
@@ -38,8 +39,8 @@ public class AzureUI extends AbstractHawkbitUI {
     AzureUI(final EventPushStrategy pushStrategy, final UIEventBus eventBus, final UIEventProvider eventProvider, final SpringViewProvider viewProvider,
             final ApplicationContext context, final DashboardMenu dashboardMenu, final ErrorView errorview,
             final NotificationUnreadButton notificationUnreadButton, final UiProperties uiProperties,
-            final VaadinMessageSource i18n) {
+            final VaadinMessageSource i18n, final ErrorHandler errorHandler) {
```

Comment:
please reference and close this PR: https://github.com/eclipse/hawkbit-extensions/pull/72/files